sys_tests/__init__.py

```diff
@@ -0,0 +1,9 @@
+from cosmo_tester.framework.testenv import bootstrap, teardown
```

Comment:
Yeah.. it had been like this initially, but then I realized that this folder took precedence before cloudify-openstack-plugin's system_tests dir on the python path, which caused problems..